### ISSUE TITLE
questions on online video object detection

### ISSUE THREAD

**Issue by gongbudaizhe** [mit-han-lab/temporal-shift-module#6]
Congratulations on the great work!

As noted in the supplementary section: "... we inserted uni-directional TSM to the backbone, while keeping other settings the same. We used the official training code of [60] to conduct the experiments".

May I ask a few questions on online video object detection:
1. How many frames are used during training? 21 frames the same as FGFA?
2. What is your learning rate policy? and optimizer? the same as FGFA?

**Comment by kuan-wang** [comment]
Hi, thanks for your interest. For your questions: 1. we use 8 frames. 2. Yes, we keep it the same as FGFA.

**Comment by gongbudaizhe** [comment]
Got it. 

One more question, please.  Since during training, only 8 frames are used. How will you apply the model for videos with frames larger than 8? You can, of course, simply ignore the discrepancy between training and testing and recurrently apply the temporal shift. Or you can divide the whole video into chunks each with 8 frames. What exactly did you do?

**Comment by tonylins** [comment]
Hi, we just ignore the discrepancy between training and testing to report the results. During testing, you just recursively replace the corresponding channels with the feature from last frame. It works quite well for online setting according to our experience.

I will close the issue for now. Feel free to re-open it if you have further questions.
